install/kubernetes/helm/istio/values.yaml

```diff
@@ -86,6 +86,40 @@ ingress:
       enabled: false
       port: 32000
 
+#
+# ingress-gateway configuration
+#
+ingress-gateway:
+  enabled: true
+  serviceAccountName: default
+  imagePullPolicy: IfNotPresent
```

Comment:
it appears we have a global image pull policy in this helm chart - recommend just using that.

Reply:
this comment was clear.  I was referring to this: https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml#L19